### ISSUE TITLE
NVDA does not descend into editable children controls in multi-line edit fields while reading

### ISSUE THREAD

**Issue by nvdaTrac** [nvaccess/test_nvdaTracImport#243]
**Reported by vtsaran on 3 Sep 2008 06:06 UTC**
**Blocked by #1668**
Re pro steps:
1. If you have Yahoo! Mail account, login into "Classic Mail" and open the list of messages.
2. Pick one of the messages, open it and press the "forward" button to open the "forward" screen.
3. Skip directly to the multi-line "message body" field, turn on "virtual buffer pass through" feature and start reading with up and down arrows.

Result: you will notice that the reading will stop right after the line that says "on [date] so and so wrote".
Pressing down arrow after this point will produce silence and nothing will be read.

Using NVDA Navigator (via numpad keys) one can find that the following text is embeded into something called "section editable"; I believe, this is actually a blockquoted section.

**Comment by nvdaTrac** [comment]
**Comment by Bernd on 22 Feb 2009 02:11 UTC**
I'm using Yahoo myself and I haven't this issue. Could you try this with NVDA 0.6p3 again? If I understand you right, you must have this issue if you write a long email, too.

**Comment by nvdaTrac** [comment]
**Comment by vtsaran on 24 Jun 2010 05:46 UTC**
I wanted to bump this up as the issue is still present even in the latest NVDA development snapshots.
The issue can also be reproduced when replying to someone else's Email. Instead of reading the original text, NVDA will say "question mark", "question mark" etc.
When inspecting the text with object navigator, the text can be seen but not when arrowing with keys.

**Comment by nvdaTrac** [comment]
**Comment by jteh on 25 Jun 2010 01:16 UTC**
I definitely haven't forgotten about this. Unfortunately, it is an extremely complex task due to the way Gecko exposes rich text. The recent merge of the compound documents code includes the basic framework that we need for this, but I haven't done the Gecko specific stuff yet.

**Comment by nvdaTrac** [comment]
**Comment by Q on 14 Apr 2013 17:00 UTC**
Bump: has this been fixed?

**Comment by nvdaTrac** [comment]
**Comment by jteh on 14 Apr 2013 21:14 UTC**
Marking as a duplicate of #1668 (the more general issue).
